packages/graph/hash_graph/lib/graph/src/lib.rs

```diff
@@ -46,6 +46,14 @@
     clippy::use_debug,
     clippy::verbose_file_reads
 )]
+// Until we do optimization work, there should be no reason to use unsafe code. When allowing unsafe
+// again don't forget to enable miri in the `Makefile.toml`. When allowing unsafe code again, we
+// should prefer `#![deny(unsafe_code)]` and only allow it in a few places unless this gets very
+// verbose.
+#![forbid(
+    unsafe_code,
+    reason = "At the current state, unsafe code should be avoided"
```

Comment:
```suggestion
    reason = "Unsafe code has been disabled until a good argument has been put forward for
     its usage"
```

Maybe we should just go with something like that? (I'm not sure it's better but I think the original reason was a bit misleading/unclear too). Ideally we just want someone to come to the lint and read the comment